helper/date_period_format_test.go

```diff
@@ -14,6 +14,11 @@ func TestDatePeriodFormat(t *testing.T) {
 		got := helper.DatePeriodFormat("2010 JUN-SEP")
 		So(got, ShouldEqual, want)
 	})
+	Convey("Given a time-series monthly string spanning two different years", t, func() {
+		want := "Dec 2010 - Jan 2011"
+		got := helper.DatePeriodFormat("2010 DEC-JAN")
```

Comment:
this format isn't want you indicated in the comments as desired:

> Further discussions has resulted in the desired output being DEC 2020 - FEB 2021 and the code added reflects that desire.

Reply:
Whatcha talkin bout willis?

Given 2010 DEC-JAN we want Dec 2010 - Jan 2011 or are you getting hung up on the capitals?

Reply:
oooooo stupid me misread. got/want (I'm used to actual/expected)